CONTRIBUTING.md

```diff
@@ -83,6 +83,9 @@ Specify the Genesys Cloud environment to target when host app is running on `loc
 #### CLIENT_APP_SDK_PC_DEV_ENVS
 Whitelist additional comma-delimited top level domains (e.g. example.com).
 
+#### CLIENT_APP_SDK_GC_EXTRA_ENVS
+Adds a list of Genesys Cloud environments that should be supported.
```

Comment:
:thumbsup: ... i was thinking about this after our chat and was going to mention we might need this.  Would you mind describing the type a bit?  maybe just Environment[] would be enough so devs will know the format of this value?